src/Infrastructure/Workflow/WorkflowRunner.php

```diff
@@ -5,12 +5,11 @@
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
 
-namespace SprykerSdk\Sdk\Infrastructure\Service;
+namespace SprykerSdk\Sdk\Infrastructure\Workflow;
```

Comment:
Currently, it can leave in the infra layer, but it contains rather business logic than infrastructure. 
In the future, the Input and Output should be extracted to the separate service and Workflow runner should be moved to the Application layer.

Reply:
we can move it later. it was refactored in API pr and other classes